assets/docs/vasudevgupta7/models/wav2vec2/1.md

```diff
@@ -0,0 +1,50 @@
+# Module vasudevgupta7/wav2vec2/1
+
+Pre-trained speech model (without any head) from Facebook for Automatic Speech Recognition
+
+<!-- asset-path:  https://huggingface.co/vasudevgupta/gsoc-wav2vec2/resolve/main/saved-model.tar.gz -->
```

Comment:
Unfortunately downloading from HuggingFace doesn't work because the compressed file is not the SavedModel folder itself, but instead it is a folder that contains the SavedModel folder. This needs to link to a compressed version of the SavedModel folder, such that the `saved_model.pb` file is immediatley accessible at the top level once the file is uncompressed.

Eg. currently uncompressing this gives output like:

```
output_location/
  /saved-model
    /assets
    /saved_model.pb
    /variables
```

Instead, it needs to look like:

```
output_location/
  /assets
  /saved_model.pb
  /variables
```

A public bucket on Google Cloud Storage is a good location for this file. It can be deleted once the model is accessible on TF Hub.

Reply:
@MorganR, I have fixed it & stored everything in GCS (https://storage.googleapis.com/wav2vec2/saved-model.tar.gz)